### PR TITLE
introducing scastie

### PR DESCRIPTION
**Pull request by MasseGuillaume**
Good to go

### CODE REVIEW

File: blog/_posts/2017-02-20-introducing-scastie.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Introducing Scastie - An interactive playground for Scala.
```
Comment (sjrd) [inline blog/_posts/2017-02-20-introducing-scastie.md]: The YAML front matter is missing. You can take example on an earlier blog post, e.g., https://github.com/scala/scala-lang/blob/master/blog/_posts/2017-01-27-first-scala-platform-meeting.md

File: blog/_posts/2017-02-20-introducing-scastie.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+We also enable a worksheet mode, which feels much like a worksheet in an IDE. This lets a user write code as top-level expressions, without having to put code inside of a class or object with a main method. Worksheet mode gives you two ways to interleave your results; on the one hand, when an expression is evaluated, you can see the value and type of the evaluated expression to the right of the code that you wrote. On the other hand, worksheet mode also makes it possible to do a kind of literate programming; that is, you may interleave code and HTML blocks much like in notebook environments like iPython notebooks.
+
+# What's next? 
+
+* Improve the sharing model. For example, we could use a similar approach as gist or JS Fiddle where you can fork and edit code snippets. 
+* Support Scala.js and Scala-Native. Vote on which one you'd like to see first! (Vote for Scala.js or vote for Scala Native)
```
Comment (sjrd) [inline blog/_posts/2017-02-20-introducing-scastie.md]: Where do we vote?

File: blog/_posts/2017-02-20-introducing-scastie.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+* Make it possible embed Scastie in your project's documentation.
```
Comment (sjrd) [inline blog/_posts/2017-02-20-introducing-scastie.md]: <ins>to</ins> embed

File: blog/_posts/2017-02-20-introducing-scastie.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+Since Scastie is a new project, we want to work out the kinks with it before everyone jumps on it and starts using it. In particular, the UI is currently a bit rough, and we're working on catching bugs in the general usage of Scastie.
+
+For that reason, we're opening up Scastie to a handful of beta users. Do you want to help us work out the kinks with Scastie? If so, head over to [https://scastie.scala-lang.org](https://scastie.scala-lang.org) to join the beta. We'll gradually be adding more users, so if you don't get immediate access, just hang on for a day or so and we'll ping you to let you know when you can try out Scastie.
+
+And remember, please give us feedback! Let us know what is confusing, if something doesn't work as expected, or if there's anything you think we can improve! There are links in Scastie back to our Gitter channels and GitHub issue tracker.
+
+# Talk to us!
+
+Thougts or opinions about Scastie? Join us over on [Scala Contributors](https://contributors.scala-lang.org) to contribute to the discussion.
```
Comment (sjrd) [inline blog/_posts/2017-02-20-introducing-scastie.md]: Typo: Thoughts

**Comment by heathermiller** [comment]
Merci @sjrd :)

**Pull request merged**